wasm-node/rsc/run_node.js

```diff
@@ -0,0 +1,44 @@
+Error.stackTraceLimit = 99;
+
+import init, { setup, Network, WasmNode, WasmNodeConfig, canonical_network_bootnodes, network_genesis } from "/wasm/wasm_node.js";
+
+// initialize wasm
+await init();
+// setup logging and console panic hook
+await setup();
```

Comment:
can we rename `setup` to `configure_logging` or setup_logging or whatever that will be more precise in what it does? I think we can treat `console_panic_hook` as part of `logging` in this case

Reply:
I was also thinking about a single `init`, that will do more stuff in the future.